CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Exposed new `dem_name` api parameter for RTC_GAMMA jobs
 
+### Changed
+- `get_job_count_for_month` now uses `Select='COUNT'` for better performance querying DynamoDB
```

Comment:
I'd be specific and call it `util.get_job_count_for_month`, maybe event `hyp3_api.util.get_job_count_for_month`